schemas/node.go

```diff
@@ -693,6 +693,7 @@ func Node() map[string]*schema.Schema {
 			Description: `user specified device name`,
 			Type:        schema.TypeString,
 			Optional:    true,
+			DiffSuppressFunc: supressNameAfterCreation("name"),
```

Comment:
According to docs I we should be able to use `computed` here. 
```
// If Computed is true, then the result of this value is computed
// (unless specified by config) on creation.
```

Reply:
But comment says `value is computed (unless specified by config) on creation`. I would interpret that if you supply name initially on creation, it will work.